_posts/2019-08-05-cognito-azuread-sso.markdown

```diff
@@ -4,7 +4,6 @@ title:  "Securing your web applications in AWS using SSO with Azure Active Direc
 date:   2019-08-05
 tags: aws cognito oidc saml2 sso azure
 author: Paul Symons
-image: 
```

Comment:
Do we need an image?

Previously these were used from the main website link.  I presume we'll re-instate that at some point.